src/Arcus.Observability.Telemetry.Core/Extensions/ILoggerExtensions.cs

```diff
@@ -31,6 +31,16 @@ public static class ILoggerExtensions
             + ContextProperties.DependencyTracking.IsSuccessful + "} - Context: {@"
             + ContextProperties.EventTracking.EventContext + "})";
 
+        private const string ServiceBusDependencyFormat =
```

Comment:
Honestly, I'm not sure why we can't re-use `DependencyFormat` if we are not adding a new converter?

Reply:
Now we use the `TargetName` which is not in the dependency format.

Reply:
Can't we add it then to our regular dependency tracking as well? Everybody should be able to specify that.

Reply:
We can make an issue of that to add the `TargetName` to that one, but what about the `EntityType` that is now added to the message format?

Reply:
That's ok, if we get rid of the SB one then or align it

Reply:
What do you mean exactly? 🤔

Reply:
Nevermind, will wait for that PR